### PR TITLE
update uninstall verb to match recent changes for --only-packages

### LINKED ISSUE #122: uninstall verb fails
broken since https://github.com/ament/ament_tools/pull/114
```
dhood@osrf-esteve:~/ros2_test_ws [ros2_test_ws]$ ament uninstall
# Topological order
Traceback (most recent call last):
  File "/home/dhood/ros2_ws/install_isolated/ament_tools/bin/ament", line 9, in <module>
    load_entry_point('ament-tools', 'console_scripts', 'ament')()
  File "/home/dhood/ros2_ws/build_isolated/ament_tools/ament_tools/commands/ament.py", line 88, in main
    rc = args.main(args)
  File "/home/dhood/ros2_ws/build_isolated/ament_tools/ament_tools/verbs/uninstall/cli.py", line 91, in main
    print_topological_order(opts, packages)
  File "/home/dhood/ros2_ws/build_isolated/ament_tools/ament_tools/verbs/build/cli.py", line 240, in print_topological_order
    if pkg_name in opts.skip_packages:
AttributeError: 'Namespace' object has no attribute 'skip_packages'
```

### PR DESCRIPTION
**Pull request by dirk-thomas**
Fixes #122.

Connect to #122.

### CODE REVIEW

File: ament_tools/verbs/uninstall/cli.py [hunk 1]
```diff
@@ -88,39 +96,45 @@ def main(opts, per_package_main=uninstall_pkg_main):
...
+                package_share = os.path.join(
+                    opts.install_space, 'share', depend)
+                opts.build_dependencies.append(package_share)
+
+            rc = per_package_callback(opts)
+            if rc:
+                return rc
         if package.name == opts.start_with:
             print("Stopped after package '{0}'".format(package.name))
             break
```
Comment (dhood) [inline ament_tools/verbs/uninstall/cli.py]: `iterate_packages` doesn't seem to always return a return code explicitly

Comment (dirk-thomas) [inline ament_tools/verbs/uninstall/cli.py]: That is right. Implicitly every function returns `None` which is a reasonable return code (becomes `0`).

Comment (dhood) [inline ament_tools/verbs/uninstall/cli.py]: ok, thank you for clarifying

**Comment by dirk-thomas** [comment]
Thanks for testing it locally.

**Pull request merged**